tests/Operation/ExplainFunctionalTest.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Driver\BulkWrite;
+use MongoDB\Operation\Count;
+use MongoDB\Operation\CreateCollection;
+use MongoDB\Operation\Distinct;
+use MongoDB\Operation\Explain;
+use MongoDB\Operation\Find;
+use MongoDB\Operation\FindAndModify;
+use MongoDB\Operation\FindOne;
+use MongoDB\Operation\InsertMany;
+
+class ExplainFunctionalTest extends FunctionalTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+        if (version_compare($this->getServerVersion(), '3.0.0', '<')) {
+            $this->markTestSkipped('Explain is not supported');
+        }
+    }
+
+    /**
+     * @dataProvider provideVerbosityInformation
+     */
+    public function testCount($verbosity, $executionStatsExpected, $allPlansExecutionExpected)
+    {
+        $insertMany = new InsertMany($this->getDatabaseName(), $this->getCollectionName(), [
+            ['x' => 0],
+            ['x' => 1],
+            ['x' => 2],
+            ['y' => 3]
+        ]);
+        $insertMany->execute($this->getPrimaryServer());
+
+        $operation = new Count($this->getDatabaseName(), $this->getCollectionName(), ['x' => ['$gte' => 1]], []);
+        $explainOperation = new Explain($this->getDatabaseName(), $operation, ['verbosity' => $verbosity, 'typeMap' => ['root' => 'array', 'document' => 'array']]);
+        $result = $explainOperation->execute($this->getPrimaryServer());
+
+        $this->assertExplainResult($result, $executionStatsExpected, $allPlansExecutionExpected);
+    }
+
+    /**
+     * @dataProvider provideVerbosityInformation
+     */
+    public function testDistinct($verbosity, $executionStatsExpected, $allPlansExecutionExpected)
+    {
+        if (version_compare($this->getServerVersion(), '3.2.0', '<')) {
+            $this->markTestSkipped('Distinct is not supported on servers with version < 3.2');
```

Comment:
"Explain command does not support distinct"

As-is, this reads as if Distinct itself isn't supported. Most of our skip messages don't repeat the version requirement, but you're welcome to add that in. Based on your change stream skip message, the following could work: "Explaining distinct command requires server version >= 3.2"